### DISCUSSION TITLE
Release discussion

### DISCUSSION THREAD

**Discussion by mattijn** [vega/altair#3065] [General]
I like to do a minor release to bump Altair to version `5.0.1` with solutions to some of the issues that came up until now (#3056, #3057, #3063) while staying on VL5.8.

**Comment by binste** [comment]
Sounds good to me! Would be great if we can use this opportunity to also publish #3060.

**Reply by mattijn** [reply]
Done, as well as https://github.com/altair-viz/altair/pull/3068.

**Reply by binste** [reply]
Thanks @mattijn!

**Comment by joelostblom** [comment]
Sounds good to me too!
